doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
+   be no database to connect the client to.
+
+Store Data
+----------
+Next, using the SmartRedis client instance, we create and store a NumPy tensor using
+``Client.put_tensor()``:
+
+.. code-block:: python
+
+    # Create NumPy array
+    array_1 = np.array([1, 2, 3, 4])
+    # Store the NumPy tensor
+    colo_client.put_tensor("tensor_1", array_1)
+
+Retrieve Data
+-------------
+Next, retrieve the tensor using ``Client.get_tensor()``:
+
+.. code-block:: python
 
-  colo_settings = exp.create_run_settings(exe="./some_mpi_app")
+    # Retrieve tensor from driver script
+    value_1 = colo_client.get_tensor("tensor_1")
+    # Log tensor
+    colo_client.log_data(LLInfo, f"The colocated db tensor is: {value_1}")
 
-  colo_model = exp.create_model("colocated_model", colo_settings)
-  colo_model.colocate_db_uds(
-          db_cpus=1,              # cpus given to the database on each node
-          debug=False             # include debug information (will be slower)
-          ifname=network_interface # specify network interface(s) to use (i.e. "ib0" or ["ib0", "lo"])
-  )
-  exp.start(colo_model)
+When the Experiment completes, you can find the following log message in `colo_model.out`::
+    Default@21-48-01:The colocated db tensor is: [1 2 3 4]
 
+The Experiment Driver Script
+============================
+To run the application, specify a Model workload from
+within the workflow (Experiment).
+Defining workflow stages requires the utilization of functions associated
+with the ``Experiment`` object.
+In this example, we instantiate an ``Experiment`` object with the name ``getting-started``.
+We setup the SmartSim ``logger`` to output information from the Experiment.
+
+.. code-block:: python
 
-By default, SmartSim will pin the database to the first _N_ CPUs according to ``db_cpus``. By
-specifying the optional argument ``custom_pinning``, an alternative pinning can be specified
-by sending in a list of CPU ids (e.g [0,2,range(5,8)]). For optimal performance, most users
-will want to also modify the RunSettings for the model to pin their application to cores not
-occupied by the database.
+    import numpy as np
+    from smartredis import Client
+    from smartsim import Experiment
+    from smartsim.log import get_logger
+    import sys
 
-.. warning::
+    exe_ex = sys.executable
+    logger = get_logger("Example Experiment Log")
+    # Initialize the Experiment
+    exp = Experiment("getting-started", launcher="auto")
 
-  Pinning is not supported on MacOS X. Setting ``custom_pinning`` to anything
-  other than ``None`` will raise a warning and the input will be ignored.
+Initialize a Colocated Model
+----------------------------
+In the next stage of the experiment, we
+create and launch a colocated ``Model`` that
+runs the application script with a database
+on the same compute node.
+
+Step 1: Configure
+"""""""""""""""""
+In this experiment, we invoke the Python interpreter to run
```

Comment:
In this experiment, we invoke the Python interpreter to run
the python script defined in section: The Application Script.
To configure this into a ``Model``, we use the ``Experiment.create_run_settings()`` function.
The function returns a ``RunSettings`` object.
A ``RunSettings`` allows you to configure
the run settings of a SmartSim entity.
We initialize a RunSettings object and
specify the path to the application file,
`application_script.py`, to the argument
``exe_args``, and the run command to ``exe``.
->

In this example experiment, the ``Model`` application is a Python script as defined in section: The Application Script.  Before creating the ``Model`` object for this application, we must use ``Experiment.create_run_settings()`` to create a ``RunSettings`` object that defines how to execute the ``Model``.  To launch the Python script in this example, we specify the path to the application file
`application_script.py` as the ``exe_args`` parameter  and the executable `exe_ex` (the Python executable on this system) as ``exe`` parameter. 
The ``Experiment.create_run_settings()`` function will return a ``RunSettings`` object that can then be used to initialize the ``Model`` object.